hazelcast-client/src/test/java/com/hazelcast/client/discovery/AwsCloudDiscoveryTest.java

```diff
@@ -44,15 +53,26 @@
 @Category({SlowTest.class, ParallelTest.class})
 public class AwsCloudDiscoveryTest {
 
+    private static final String ACCESS_KEY = AwsProperties.ACCESS_KEY.getDefinition().key();
+    private static final String SECRET_KEY = AwsProperties.SECRET_KEY.getDefinition().key();
+
     @Test
     public void testAwsClient_MemberNonDefaultPortConfig() {
         Map<String, Comparable> props = new HashMap<String, Comparable>();
-        props.put(AwsProperties.PORT.getDefinition().key(), "60000");
-        props.put(AwsProperties.ACCESS_KEY.getDefinition().key(), System.getenv("AWS_ACCESS_KEY_ID"));
-        props.put(AwsProperties.SECRET_KEY.getDefinition().key(), System.getenv("AWS_SECRET_ACCESS_KEY"));
-        props.put(AwsProperties.TAG_KEY.getDefinition().key(), "aws-test-tag");
-        props.put(AwsProperties.TAG_VALUE.getDefinition().key(), "aws-tag-value-1");
-        props.put(AwsProperties.CONNECTION_TIMEOUT_SECONDS.getDefinition().key(), "10");
+        props.put(PORT.getDefinition().key(), "60000");
+        props.put(ACCESS_KEY, System.getenv("AWS_ACCESS_KEY_ID"));
+        props.put(SECRET_KEY, System.getenv("AWS_SECRET_ACCESS_KEY"));
+        props.put(TAG_KEY.getDefinition().key(), "aws-test-tag");
+        props.put(TAG_VALUE.getDefinition().key(), "aws-tag-value-1");
+        props.put(CONNECTION_TIMEOUT_SECONDS.getDefinition().key(), "10");
+
+        if (isOnJenkins()) {
```

Comment:
Here we distinct to either fail or ignore the test if the environment variables are not set, depending if we are running on Jenkins or not.